bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,246 @@ class StylintBear:
+            When ``True``, expect ``:`` when declaring a property. When
+            ``False``, expect no ``:`` when declaring a property.
+        :param color_variables_for_hex_values:
+            When ``True``, enforce variables when defining hex values.
+        :param spaces_after_commas:
+            Enforce or disallow spaces after commas.
+        :param spaces_after_comments:
+            Enforce or disallow spaces after line comments.
+            For example: If set to ``True``, prefer
+            ``// comment`` over ``//comment``.
+        :param allow_trailing_whitespace:
+            If ``True``, ignores trailing whitespace. If ``False``, trailing
+            whitespace will throw a warning.
+        :param no_css_literals:
+            By default Stylint ignores ``@css`` blocks. If set to ``True``
+            however, warnings are thrown if ``@css`` is used.
+        :param max_selector_depth:
+            Set the max selector depth. If set to 4, max selector depth will
+            be 4 indents. Pseudo selectors like ``&:first-child`` or
+            ``&:hover`` won't count towards the limit.
+        :param check_duplicates:
+            Checks if selectors or properties are duplicated unnecessarily.
+        :param efficient_properties:
+            Check for places where properties can be written more efficiently.
+        :param extend_preference:
+            Pass in either ``@extend`` or ``@extends`` and then enforce that.
+            Both are valid in Stylus. It doesn't really matter which one
+            you use.
+            For example: If set to ``@extends``, prefer
+            ``@extends $some-var`` instead of ``@extend $some-var``
+            or if set to ``@extend``, prefer
+            ``@extend $some-var`` over ``@extends $some-var``.
+        :param indent_size:
+            This works in conjunction with ``max_selector_depth``. If you
+            indent with spaces this is the number of spaces you indent with.
+            If you use hard tabs, set this value to ``0``.
+            For example: If set to ``2``, prefer
+            ``/s/smargin: 0`` over ``/s/s/smargin: 0``.
+        :param leading_zero:
+            When ``True``, prefer leading zeroes on decimal points. When
+            ``False``, disallow leading zeroes on decimal points.
+        :param max_errors:
+            Set maximum number of errors. If ``0``, all errors will
+            be shown without a limit.
+        :param max_warnings:
+            Set maximum number of warnings. If ``0``, all errors will
+            be shown without a limit.
+        :param mixed_spaces_and_tabs:
+            If a non-negative number is passed to ``indent_size``,
+            soft tabs (i.e. spaces) are assumed, and if
+            ``0`` is passed to ``indent_size``, hard tabs are assumed.
+            For example: If ``indent_size = 4`` and
+            ``mixed_spaces_and_tabs = True``, prefer
+            ``s\s\s\smargin\s0`` over ``\tmargin\s0``
```

Comment:
Btw you use normal slashes above, maybe you should here too^^

(You only have to show the first spaces, not the ones in the middle imo: `/s/s/s/smargin 0` instead of `/s/s/s/smargin/s0`)